CODEOWNERS

```diff
@@ -1,6 +1,8 @@
 /cpp/ @github/codeql-c-analysis
 /cpp/autobuilder/ @github/codeql-c-extractor
```

Comment:
```suggestion
```
as the folder will cease to exist.

Reply:
Well in theory the README file stays there, but you're right, it doesn't require a separate CODEOWNERS entry. I'll delete the line.